renderer/src/components/CharacterKillsCard.jsx

```
@@ -0,0 +1,170 @@
+            </div>
+          </div>
+        </div>
+      )}
+      
+      <div className="p-4">
+        {/* Main action button */}
+        <button
+          onClick={fetchAllCharacterKills}
+          disabled={loading}
+          className="w-full px-4 py-3 bg-indigo-600 text-white rounded-md hover:bg-indigo-700 transition-colors disabled:opacity-50 flex items-center justify-center space-x-2 mb-4"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)